runtime/compiler/trj9/optimizer/JProfilingValue.cpp

```diff
@@ -264,34 +264,56 @@ void
 TR_JProfilingValue::lowerCalls()
    {
    TR::TreeTop *cursor = comp()->getStartTree();
+   int32_t numberOfNodes = comp()->getFlowGraph()->getNextNodeNumber();
+   TR_BitVector *backwardAnalyzedListOfAddressNodes = new (comp()->trStackMemory())TR_BitVector(numberOfNodes, trMemory(), stackAlloc);
```

Comment:
Isn't the number of nodes going to grow in the loop below if you end up uncommoning a node? @andrewcraik will this break things on a subsequent iteration?

Reply:
This should be allocated growable and should just grow as needed to save space rather than zeroing everything. If it is growable and not presized the increase in nodes won't hurt.